Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -30,167 +30,320 @@
 #import "LoggingNative.h"
 #import "NSOperationInternal.h"
 
+#include <deque>
+#include <mutex>
+#include <condition_variable>
+#include <algorithm>
+#include <memory>
+
 static const wchar_t* TAG = L"NSOperationQueue";
 
-typedef void* gpointer;
+class NSOperationQueuePriv {
+public:
+    NSOperationQueuePriv(NSOperationQueue* parent) : _thread(nullptr), _maxConcurrentOperationCount(1), _isSuspended(NO) {
```

Comment:
True. I just kept the priv around as the encapsulating doodad but that could have just as easily been the objc class itself.

---
In reply to: [95450433](https://github.com/Microsoft/WinObjC/pull/1676#discussion_r95450433) [](ancestors = 95450433)